app/experimenter/experiments/forms.py

```diff
@@ -1134,6 +1134,13 @@ class Meta:
     def clean_archived(self):
         return not self.instance.archived
 
+    def save(self, *args, **kwargs):
+        experiment = super().save(*args, **kwargs)
+        tasks.update_bug_resolution_task.delay(
```

Comment:
We need to update the tests for this to mock out this delay call and make sure it gets called